pages/home.md

```diff
@@ -14,8 +14,8 @@ permalink: /
         />
       </figure>
       <p class="subtitle has-text-centered is-size-4 is-size-3-tablet is-size-2-desktop">
-      Developing Web standards that
-      empower people to take control of their data. 
+        Developing Web standards that empower people to take control of their
+        data, including choosing how your data is used.
```

Comment:
@RubenVerborgh  - wasn't sure whether to go with:

- "choosing how your data is used"
- or a more specific "choosing the applications that can access your data"
- or whether a different choice all together.

(Note: I did in the Pod section below mention the choosing the applications)

Reply:
At this point I feel like the previous version is most viable, possibly with @kay-kim's suggestion of changing "Using" to "Developing". So change: "Using Web standards to let people control their data, and choose the applications and services to use with it." to "Developing Web standards to let people control their data, and choose the applications and services to use with it."

Reply:
I'm good with whichever you all decide (as I'm sure, we'll refresh this page as the project moves along - and people's familiarity with/awareness of Solid changes).

Reply:
@justinwb / @RubenVerborgh --
trying out couple of options from this thread and [this other thread](https://github.com/solid/solidproject.org/pull/475/files#r554880483) - (Apologies for the wonky review with comments everywhere - am used to a different tool for reviews, so wasn't sure the rules for resolving, commenting, etc.)

 The middle button to Ongoing Work links to https://solid.github.io/specification/ page.

<img width="1271" alt="Screen Shot 2021-01-11 at 2 27 41 PM" src="https://user-images.githubusercontent.com/2126636/104228742-48f71500-5419-11eb-91d9-95828edb4332.png">

<img width="1455" alt="Screen Shot 2021-01-11 at 2 24 58 PM" src="https://user-images.githubusercontent.com/2126636/104228740-485e7e80-5419-11eb-9585-4669ac0ea26f.png">

Reply:
Oh .. and not wedded to having all three buttons either.

Reply:
drat - missed mentioning Solid up here at the blurb.  Well, for now, imagine that we spell out Solid on these two and once we decide on something (could be none of the above), will make sure to include Solid: in whatever we choose.

Reply:
There's potential in the first. How about we turn it around?

> Your data, your choices.
> Advancing Web standards to empower people.

And prefix either of those lines with "Solid: "?
Because only we know what the S means.


The second is too long for me.

Reply:
<img width="1114" alt="Screen Shot 2021-01-11 at 2 59 55 PM" src="https://user-images.githubusercontent.com/2126636/104232062-c4f35c00-541d-11eb-970b-2a10bc5b039b.png">

<img width="979" alt="Screen Shot 2021-01-11 at 3 00 19 PM" src="https://user-images.githubusercontent.com/2126636/104232064-c58bf280-541d-11eb-9dc6-64f09ee946a5.png">

Reply:
The preview helps! I like 2.

Reply:
Will submit a patch with #2 and give Justin a chance to weigh in.

Reply:
patch is up now (squashed and rebased just to make future merge easier).

Reply:
Heh -- apparently, I really wasn't wedded to having the 3 buttons:
- Put back the Get a Pod
- Removed the About Solid -- realized that page already had 2 links to About
- Removed the Ongoing Work. 

Ultimately, what I would like is each section below have a button that takes people somewhere interesting.  So, my latest patch is below (where I added a button to the specification repo  in the interoperability section).  Later on, would like to rework the sections we show here (as well as rework the bottom "Solid creates ..." panel into a section and have that go somewhere else) once we have enough pages for people to land.

<img width="693" alt="Screen Shot 2021-02-03 at 10 56 30 PM" src="https://user-images.githubusercontent.com/2126636/106843085-33b98300-6673-11eb-9104-8bf984f1c832.png">

Reply:
Agree with @RubenVerborgh on the second option. Only thought I have on the tagline is whether to make "choices" singular -> "choice". I'm not opposed to plural but wonder if singular carries more weight as something you do, vs. something you've done.

A few additional thoughts / suggestions:

- The first about block was adjusted and the get a pod action item is repeated. Our top problem has historically not been that people can't get pods, but that people don't know what Solid is. I think we want this section to focus on two specific destinations: 1) Learn More (linking to about). 2) Read the specification (linking to https://solidproject.org/TR/) - note that any links to spec documentation should go there now rather than solid.github.io.

- Considering moving the "Get your own pod" under "Store anything" - the current Application list needs to be vetting. Many of those are broken, so we were avoiding the spotlight until they're cleaned up. Could also opt to have no action there for now.

- _Read the Interoperability Spec and Other Standards_ feels like a lot to text to pack into that button. Maybe just "Read the Specifications"?

Reply:
Thanks @justinwb.  

- Choice vs Choices -- I'm also okay with either one.
- About the Learn More ... and linking to About Page... (And thanks for heads up on the new link to TR 👍 ) 
  -  I'm kind of thinking that the landing page should quickly cover what Solid is as opposed to having readers go to the About page immediately above the fold.  Because to have people go to another page immediately minimizes the value of this page (that is, why even have users land on this page).  
  - I guess I'm also curious if the About page is actually answering the question (I mean, even w/o the button - we link to it 3 times on that page).  That is, I kind of feel like just having the links to the About page and the specs ... they still kind of skirt around the issue of having our readers infer or deduce what it is rather than explicitly state what Solid is.  For me personally, I think of Solid in two ways (these may be a bit simplistic, but eh ... 😄 - probably should mention a bit about Solid applications in the ecosystem section but am spitballing here at the moment) 
  - Solid is a project that envisions an ecosystem where
     - people store their data in Pods.
     - people can decide which applications can access their data. They can also revoke access to their data.
  - Underlying this ecosystem are the Solid specifications. These determine the behavior of Pods and applications such that people can use their data seamlessly across different applications and services.
  
- The "Store anything"  link to the Apps ... Heh, am totally on board with you w.r.t. the state of the Applications page. but, although the Applications page needs some care, it did seem like after people get a Pod, we kind of leave people hanging with "Now what."  (Also, thought maybe having the link may cause us to start cleaning up the App page in my passive-aggressive way 😄 )  But it could be that if we have the first section talk about Solid (what it is), then can defiinitely work in the Pod and store anything content together.
-  👍 on shortening the button caption to Read the Specifications.

Reply:
So ... I took a whack at adding some kind of definition of Solid to the first section and then merged the two Pod sections into one.  We could actually move the "Underlying ..." paragraph out from the first section - but wanted to give a general idea about defining Solid on the first page.
(updated the Specifications link - both text and target).

https://deploy-preview-475--solidproject-dot-org.netlify.app/